fli-iam/shanoir-ng
# Solr : selection list + deletion

* selection list, what to test
    * select datasets
    * navigate between the 2 tabs : selection list and result list
    * edit search while in selection -> switches to results
    * navigating back from another page should restore saved selection
    * test "clear selection" button
* delete feature
    * try to delete datasets not in a study the current user administrates (forbidden)
    * try to mix authorized and forbidden datasets
...
